### ISSUE TITLE
Stored opencaching.de caches not refreshing

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#14501]
### Describe your problem!

Three users contacted us and are reporting problem to refresh OC caches.
Its not reproducible for me and also for the users it does not happen to all OC caches.

Affected users are on 2023.06.18 and on 2023.08.05-RC
Support tickets 209279, 948696

### How to reproduce?

- Open a stored OC.de cache
- Trigger refresh

### Actual result after these steps?

- Refresh reports finished
- c:geo still states "Refreshed x weeks ago"
- Data has not been changed (e.g. logbook still missing new entries)

### Expected result after these steps?

- Normal refresh

### Reproducible

Yes

### c:geo Version

2023.06.18

**Comment by eddiemuc** [comment]
Example of oc caches where bug occured and log file might be helpful. Is the problem reproducible for same cache?

**Comment by Lineflyer** [comment]
I just now got a debug log of such a failed refresh.
Communication with OKAPI looks IMHO normal, but we fail to parse the cache:

```
08-12 13:56:19.558 16032 25898 E cgeo    : [refresh--1] OkapiClient.parseCache {OkapiClient.parseCache:802/OkapiClient.getCache:232/OCApiConnector.searchByGeocode:79/Geocache.searchByGeocode:2158/Geocache.storeCache:2058/Geocache.refreshSynchronous:2027/Geocache.refreshSynchronous:2021/Geocache.lambda$refresh$4:2017}
08-12 13:56:19.558 16032 25898 E cgeo    : java.lang.NullPointerException: Attempt to invoke interface method 'boolean java.util.List.isEmpty()' on a null object reference
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.models.Waypoint.mergeWayPoints(Waypoint.java:139)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.models.Geocache.mergeWaypoints(Geocache.java:412)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.connector.oc.OkapiClient.parseCache(OkapiClient.java:785)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.connector.oc.OkapiClient.getCache(OkapiClient.java:232)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.connector.oc.OCApiConnector.searchByGeocode(OCApiConnector.java:79)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.models.Geocache.searchByGeocode(Geocache.java:2158)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.models.Geocache.storeCache(Geocache.java:2058)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.models.Geocache.refreshSynchronous(Geocache.java:2027)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.models.Geocache.refreshSynchronous(Geocache.java:2021)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.models.Geocache.lambda$refresh$4(Geocache.java:2017)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.models.Geocache.$r8$lambda$DZTiYZWuYZA0hxIqUWOy-0OdhP8(Unknown Source:0)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.models.Geocache$$ExternalSyntheticLambda1.run(Unknown Source:4)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at io.reactivex.rxjava3.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:38)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at io.reactivex.rxjava3.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:25)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at java.lang.Thread.run(Thread.java:1012)
```

Any clue yet or more info needed?

Here is the complete sequence:
```
08-12 13:56:19.483 16032 25898 D cgeo    : [refresh--1] HTTP-GET: https://www.opencaching.de/okapi/services/caches/geocache?attribution_append=none&cache_code=OC17B2E&fields=code%7Cname%7Clocation%7Ctype%7Cstatus%7Cdifficulty%7Cterrain%7Csize%7Csize2%7Cdate_hidden%7Ctrackables_count%7Cowner%7Cfounds%7Cnotfounds%7Crating%7Crating_votes%7Crecommendations%7Cregion%7Ccountry2%7Cattr_acodes%7Cattrnames%7Cdescription%7Chint%7Cimages%7Clatest_logs%7Calt_wpts%7Creq_passwd%7Ctrackables%7Cis_found%7Cis_recommended%7Cmy_notes%7Cgc_code%7Cattribution_note%7Cwillattends%7Cshort_description&langpref=de%7Cen&log_fields=uuid%7Cdate%7Cuser%7Ctype%7Ccomment%7Cimages%7Cinternal_id&lpc=all&oauth_consumer_key=J6M7x5rNz9YFJX38fE9n&oauth_nonce=524dfe4baba92630b007666318cdb340&oauth_signature_method=HMAC-SHA1&oauth_timestamp=1691841379&oauth_token=MWCsJ4VtsjGfQG3UuDv6&oauth_version=1.0&oauth_signature=U3Q%2FOGUVXPgxxbbOGvKp267eWKA%3D {Network.request:361/Network.getRequest:489/OkapiClient.request:1229/OkapiClient.getRequest:1240/OkapiClient.getCache:230/OCApiConnector.searchByGeocode:79/Geocache.searchByGeocode:2158/Geocache.storeCache:2058}
08-12 13:56:19.483 16032 25891 D cgeo    : [OkHttp] GET https://www.opencaching.de/okapi/services/caches/geocache?attribution_append=none&cache_code=OC17B2E&fields=code%7Cname%7Clocation%7Ctype%7Cstatus%7Cdifficulty%7Cterrain%7Csize%7Csize2%7Cdate_hidden%7Ctrackables_count%7Cowner%7Cfounds%7Cnotfounds%7Crating%7Crating_votes%7Crecommendations%7Cregion%7Ccountry2%7Cattr_acodes%7Cattrnames%7Cdescription%7Chint%7Cimages%7Clatest_logs%7Calt_wpts%7Creq_passwd%7Ctrackables%7Cis_found%7Cis_recommended%7Cmy_notes%7Cgc_code%7Cattribution_note%7Cwillattends%7Cshort_description&langpref=de%7Cen&log_fields=uuid%7Cdate%7Cuser%7Ctype%7Ccomment%7Cimages%7Cinternal_id&lpc=all&oauth_consumer_key=J6M7x5rNz9YFJX38fE9n&oauth_nonce=524dfe4baba92630b007666318cdb340&oauth_signature_method=HMAC-SHA1&oauth_timestamp=1691841379&oauth_token=MWCsJ4VtsjGfQG3UuDv6&oauth_version=1.0&oauth_signature=U3Q%2FOGUVXPgxxbbOGvKp267eWKA%3D {Network$LoggingInterceptor.intercept:402/RealInterceptorChain.proceed:109/Network$HeadersInterceptor.intercept:390/RealInterceptorChain.proceed:109/RealCall.getResponseWithInterceptorChain$okhttp:201/RealCall$AsyncCall.run:517/ThreadPoolExecutor.runWorker:1137/ThreadPoolExecutor$Worker.run:637}
08-12 13:56:19.485 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]: performTraversals params={(0,0)(wrapxwrap) gr=CENTER sim={adjust=pan forwardNavigation} ty=APPLICATION fmt=TRANSLUCENT wanim=0x103030a surfaceInsets=Rect(120, 120 - 120, 120)
08-12 13:56:19.485 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]:   fl=1820082
08-12 13:56:19.485 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]:   pfl=42000000
08-12 13:56:19.485 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]:   bhv=DEFAULT
08-12 13:56:19.485 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]:   fitTypes=STATUS_BARS NAVIGATION_BARS CAPTION_BAR
08-12 13:56:19.485 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]:  dimAmount=0.6 naviIconColor=0}
08-12 13:56:19.485 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]: performTraversals mFirst=true windowShouldResize=false viewVisibilityChanged=false mForceNextWindowRelayout=false params={(0,0)(wrapxwrap) gr=CENTER sim={adjust=pan forwardNavigation} ty=APPLICATION fmt=TRANSLUCENT wanim=0x103030a surfaceInsets=Rect(120, 120 - 120, 120)
08-12 13:56:19.485 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]:   fl=1820082
08-12 13:56:19.485 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]:   pfl=42000000
08-12 13:56:19.485 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]:   bhv=DEFAULT
08-12 13:56:19.485 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]:   fitTypes=STATUS_BARS NAVIGATION_BARS CAPTION_BAR
08-12 13:56:19.485 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]:  dimAmount=0.6 naviIconColor=0}
08-12 13:56:19.489 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]: updateBlastSurfaceIfNeeded mBlastBufferQueue=null isSameSurfaceControl=false
08-12 13:56:19.490 16032 16032 I BLASTBufferQueue: new BLASTBufferQueue, mName= ViewRootImpl@ed1de90[Aktualisieren] mNativeObject= 0xb4000076ae6b6a00 sc.mNativeObject= 0xb4000076af79a840 caller= android.view.ViewRootImpl.updateBlastSurfaceIfNeeded:2909 android.view.ViewRootImpl.relayoutWindow:9847 android.view.ViewRootImpl.performTraversals:3884 android.view.ViewRootImpl.doTraversal:3116 android.view.ViewRootImpl$TraversalRunnable.run:10885 android.view.Choreographer$CallbackRecord.run:1301 android.view.Choreographer$CallbackRecord.run:1309 android.view.Choreographer.doCallbacks:923 android.view.Choreographer.doFrame:852 android.view.Choreographer$FrameDisplayEventReceiver.run:1283 
08-12 13:56:19.490 16032 16032 I BLASTBufferQueue: update, w= 1266 h= 891 mName = ViewRootImpl@ed1de90[Aktualisieren] mNativeObject= 0xb4000076ae6b6a00 sc.mNativeObject= 0xb4000076af79a840 format= -3 caller= android.graphics.BLASTBufferQueue.<init>:84 android.view.ViewRootImpl.updateBlastSurfaceIfNeeded:2909 android.view.ViewRootImpl.relayoutWindow:9847 android.view.ViewRootImpl.performTraversals:3884 android.view.ViewRootImpl.doTraversal:3116 android.view.ViewRootImpl$TraversalRunnable.run:10885 
08-12 13:56:19.490 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]: Relayout returned: old=(0,88,1080,2256) new=(27,846,1053,1497) req=(1026,651)0 dur=4 res=0x3 s={true 0xb40000774798c800} ch=true seqId=0
08-12 13:56:19.490 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]: mThreadedRenderer.initialize() mSurface={isValid=true 0xb40000774798c800} hwInitialized=true
08-12 13:56:19.490 16032 16091 D OpenGLRenderer: eglCreateWindowSurface
08-12 13:56:19.491 16032 16032 D ScrollView:  onsize change changed 
08-12 13:56:19.492 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]: reportNextDraw android.view.ViewRootImpl.performTraversals:4438 android.view.ViewRootImpl.doTraversal:3116 android.view.ViewRootImpl$TraversalRunnable.run:10885 android.view.Choreographer$CallbackRecord.run:1301 android.view.Choreographer$CallbackRecord.run:1309 
08-12 13:56:19.492 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]: Setup new sync id=0
08-12 13:56:19.492 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]: Setting syncFrameCallback
08-12 13:56:19.492 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]: registerCallbacksForSync syncBuffer=false
08-12 13:56:19.493 16032 16143 I ViewRootImpl@ed1de90[Aktualisieren]: Received frameDrawingCallback syncResult=0 frameNum=1.
08-12 13:56:19.493 16032 16143 I ViewRootImpl@ed1de90[Aktualisieren]: Setting up sync and frameCommitCallback
08-12 13:56:19.495 16032 16091 I BLASTBufferQueue: [ViewRootImpl@ed1de90[Aktualisieren]#28](f:0,a:0) onFrameAvailable the first frame is available
08-12 13:56:19.495 16032 16091 I ViewRootImpl@ed1de90[Aktualisieren]: Received frameCommittedCallback lastAttemptedDrawFrameNum=1 didProduceBuffer=true
08-12 13:56:19.495 16032 16091 D OpenGLRenderer: CFMS:: SetUp Pid : 16032    Tid : 16091
08-12 13:56:19.495 16032 16091 W Parcel  : Expecting binder but got null!
08-12 13:56:19.495 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]: onSyncComplete
08-12 13:56:19.496 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]: setupSync seqId=0 mSyncId=0 fn=1 caller=android.view.ViewRootImpl$$ExternalSyntheticLambda11.accept:6 android.window.SurfaceSyncer.lambda$setupSync$1$android-window-SurfaceSyncer:128 android.window.SurfaceSyncer$$ExternalSyntheticLambda1.accept:8 android.window.SurfaceSyncer$SyncSet.checkIfSyncIsComplete:382 android.window.SurfaceSyncer$SyncSet.markSyncReady:359 android.window.SurfaceSyncer.markSyncReady:151 android.view.ViewRootImpl.performTraversals:4503 
08-12 13:56:19.496 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]: reportDrawFinished seqId=0 mSyncId=-1 fn=1 mSurfaceChangedTransaction=0xb400007747bd3000
08-12 13:56:19.504 16032 16032 I ViewRootImpl@acd025c[CacheDetailActivity]: MSG_WINDOW_FOCUS_CHANGED 0 0
08-12 13:56:19.550 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]: MSG_WINDOW_FOCUS_CHANGED 1 0
08-12 13:56:19.550 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]: mThreadedRenderer.initializeIfNeeded()#2 mSurface={isValid=true 0xb40000774798c800}
08-12 13:56:19.552 16032 25891 D cgeo    : [OkHttp] 200 (69 ms) GET https://www.opencaching.de/okapi/services/caches/geocache?attribution_append=none&cache_code=OC17B2E&fields=code%7Cname%7Clocation%7Ctype%7Cstatus%7Cdifficulty%7Cterrain%7Csize%7Csize2%7Cdate_hidden%7Ctrackables_count%7Cowner%7Cfounds%7Cnotfounds%7Crating%7Crating_votes%7Crecommendations%7Cregion%7Ccountry2%7Cattr_acodes%7Cattrnames%7Cdescription%7Chint%7Cimages%7Clatest_logs%7Calt_wpts%7Creq_passwd%7Ctrackables%7Cis_found%7Cis_recommended%7Cmy_notes%7Cgc_code%7Cattribution_note%7Cwillattends%7Cshort_description&langpref=de%7Cen&log_fields=uuid%7Cdate%7Cuser%7Ctype%7Ccomment%7Cimages%7Cinternal_id&lpc=all&oauth_consumer_key=J6M7x5rNz9YFJX38fE9n&oauth_nonce=524dfe4baba92630b007666318cdb340&oauth_signature_method=HMAC-SHA1&oauth_timestamp=1691841379&oauth_token=MWCsJ4VtsjGfQG3UuDv6&oauth_version=1.0&oauth_signature=U3Q%2FOGUVXPgxxbbOGvKp267eWKA%3D (http/1.1) {Network$LoggingInterceptor.intercept:410/RealInterceptorChain.proceed:109/Network$HeadersInterceptor.intercept:390/RealInterceptorChain.proceed:109/RealCall.getResponseWithInterceptorChain$okhttp:201/RealCall$AsyncCall.run:517/ThreadPoolExecutor.runWorker:1137/ThreadPoolExecutor$Worker.run:637}
08-12 13:56:19.558 16032 25898 E cgeo    : [refresh--1] OkapiClient.parseCache {OkapiClient.parseCache:802/OkapiClient.getCache:232/OCApiConnector.searchByGeocode:79/Geocache.searchByGeocode:2158/Geocache.storeCache:2058/Geocache.refreshSynchronous:2027/Geocache.refreshSynchronous:2021/Geocache.lambda$refresh$4:2017}
08-12 13:56:19.558 16032 25898 E cgeo    : java.lang.NullPointerException: Attempt to invoke interface method 'boolean java.util.List.isEmpty()' on a null object reference
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.models.Waypoint.mergeWayPoints(Waypoint.java:139)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.models.Geocache.mergeWaypoints(Geocache.java:412)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.connector.oc.OkapiClient.parseCache(OkapiClient.java:785)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.connector.oc.OkapiClient.getCache(OkapiClient.java:232)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.connector.oc.OCApiConnector.searchByGeocode(OCApiConnector.java:79)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.models.Geocache.searchByGeocode(Geocache.java:2158)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.models.Geocache.storeCache(Geocache.java:2058)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.models.Geocache.refreshSynchronous(Geocache.java:2027)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.models.Geocache.refreshSynchronous(Geocache.java:2021)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.models.Geocache.lambda$refresh$4(Geocache.java:2017)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.models.Geocache.$r8$lambda$DZTiYZWuYZA0hxIqUWOy-0OdhP8(Unknown Source:0)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at cgeo.geocaching.models.Geocache$$ExternalSyntheticLambda1.run(Unknown Source:4)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at io.reactivex.rxjava3.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:38)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at io.reactivex.rxjava3.internal.schedulers.ScheduledDirectTask.call(ScheduledDirectTask.java:25)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
08-12 13:56:19.558 16032 25898 E cgeo    : 	at java.lang.Thread.run(Thread.java:1012)
08-12 13:56:19.561 16032 16130 D cgeo    : [network--6] HTTP-GET: https://www.opencaching.de/resource2/ocstyle/images/attributes/safari.png {Network.request:361/Network.getRequest:477/HtmlImage.downloadOrRefreshCopy:349/HtmlImage.access$200:60/HtmlImage$1.downloadAndSave:286/HtmlImage$1.lambda$subscribe$0:267/HtmlImage$1.$r8$lambda$jVKmUDgXJYcnuEe2u_jpYzXaRxY:0/HtmlImage$1$$ExternalSyntheticLambda2.run:6}
08-12 13:56:19.561 16032 25891 D cgeo    : [OkHttp] GET https://www.opencaching.de/resource2/ocstyle/images/attributes/safari.png {Network$LoggingInterceptor.intercept:402/RealInterceptorChain.proceed:109/Network$HeadersInterceptor.intercept:390/RealInterceptorChain.proceed:109/RealCall.getResponseWithInterceptorChain$okhttp:201/RealCall$AsyncCall.run:517/ThreadPoolExecutor.runWorker:1137/ThreadPoolExecutor$Worker.run:637}
08-12 13:56:19.572 16032 16135 D cgeo    : [network--7] HTTP-GET: https://www.opencaching.de/images/uploads/2DBAC6B1-2ABB-11EE-8B48-CDF35CEF9D44.jpg {Network.request:361/Network.getRequest:477/HtmlImage.downloadOrRefreshCopy:349/HtmlImage.access$200:60/HtmlImage$1.downloadAndSave:286/HtmlImage$1.lambda$subscribe$0:267/HtmlImage$1.$r8$lambda$jVKmUDgXJYcnuEe2u_jpYzXaRxY:0/HtmlImage$1$$ExternalSyntheticLambda2.run:6}
08-12 13:56:19.572 16032 25882 D cgeo    : [OkHttp] GET https://www.opencaching.de/images/uploads/2DBAC6B1-2ABB-11EE-8B48-CDF35CEF9D44.jpg {Network$LoggingInterceptor.intercept:402/RealInterceptorChain.proceed:109/Network$HeadersInterceptor.intercept:390/RealInterceptorChain.proceed:109/RealCall.getResponseWithInterceptorChain$okhttp:201/RealCall$AsyncCall.run:517/ThreadPoolExecutor.runWorker:1137/ThreadPoolExecutor$Worker.run:637}
08-12 13:56:19.576 16032 16140 D cgeo    : [network--10] HTTP-GET: https://www.opencaching.de/images/uploads/3FAD6F38-2ABB-11EE-8B48-CDF35CEF9D44.jpg {Network.request:361/Network.getRequest:477/HtmlImage.downloadOrRefreshCopy:349/HtmlImage.access$200:60/HtmlImage$1.downloadAndSave:286/HtmlImage$1.lambda$subscribe$0:267/HtmlImage$1.$r8$lambda$jVKmUDgXJYcnuEe2u_jpYzXaRxY:0/HtmlImage$1$$ExternalSyntheticLambda2.run:6}
08-12 13:56:19.576 16032 25890 D cgeo    : [OkHttp] GET https://www.opencaching.de/images/uploads/3FAD6F38-2ABB-11EE-8B48-CDF35CEF9D44.jpg {Network$LoggingInterceptor.intercept:402/RealInterceptorChain.proceed:109/Network$HeadersInterceptor.intercept:390/RealInterceptorChain.proceed:109/RealCall.getResponseWithInterceptorChain$okhttp:201/RealCall$AsyncCall.run:517/ThreadPoolExecutor.runWorker:1137/ThreadPoolExecutor$Worker.run:637}
08-12 13:56:19.576 16032 16137 D cgeo    : [network--8] HTTP-GET: https://www.opencaching.de/images/uploads/4D9C9129-2ABB-11EE-8B48-CDF35CEF9D44.jpg {Network.request:361/Network.getRequest:477/HtmlImage.downloadOrRefreshCopy:349/HtmlImage.access$200:60/HtmlImage$1.downloadAndSave:286/HtmlImage$1.lambda$subscribe$0:267/HtmlImage$1.$r8$lambda$jVKmUDgXJYcnuEe2u_jpYzXaRxY:0/HtmlImage$1$$ExternalSyntheticLambda2.run:6}
08-12 13:56:19.577 16032 25891 D cgeo    : [OkHttp] 304 [Not Modified] (15 ms) GET https://www.opencaching.de/resource2/ocstyle/images/attributes/safari.png (http/1.1) {Network$LoggingInterceptor.intercept:412/RealInterceptorChain.proceed:109/Network$HeadersInterceptor.intercept:390/RealInterceptorChain.proceed:109/RealCall.getResponseWithInterceptorChain$okhttp:201/RealCall$AsyncCall.run:517/ThreadPoolExecutor.runWorker:1137/ThreadPoolExecutor$Worker.run:637}
08-12 13:56:19.577 16032 25901 D cgeo    : [OkHttp] GET https://www.opencaching.de/images/uploads/4D9C9129-2ABB-11EE-8B48-CDF35CEF9D44.jpg {Network$LoggingInterceptor.intercept:402/RealInterceptorChain.proceed:109/Network$HeadersInterceptor.intercept:390/RealInterceptorChain.proceed:109/RealCall.getResponseWithInterceptorChain$okhttp:201/RealCall$AsyncCall.run:517/ThreadPoolExecutor.runWorker:1137/ThreadPoolExecutor$Worker.run:637}
08-12 13:56:19.590 16032 25882 D cgeo    : [OkHttp] 304 [Not Modified] (18 ms) GET https://www.opencaching.de/images/uploads/2DBAC6B1-2ABB-11EE-8B48-CDF35CEF9D44.jpg (http/1.1) {Network$LoggingInterceptor.intercept:412/RealInterceptorChain.proceed:109/Network$HeadersInterceptor.intercept:390/RealInterceptorChain.proceed:109/RealCall.getResponseWithInterceptorChain$okhttp:201/RealCall$AsyncCall.run:517/ThreadPoolExecutor.runWorker:1137/ThreadPoolExecutor$Worker.run:637}
08-12 13:56:19.590 16032 25890 D cgeo    : [OkHttp] 304 [Not Modified] (14 ms) GET https://www.opencaching.de/images/uploads/3FAD6F38-2ABB-11EE-8B48-CDF35CEF9D44.jpg (http/1.1) {Network$LoggingInterceptor.intercept:412/RealInterceptorChain.proceed:109/Network$HeadersInterceptor.intercept:390/RealInterceptorChain.proceed:109/RealCall.getResponseWithInterceptorChain$okhttp:201/RealCall$AsyncCall.run:517/ThreadPoolExecutor.runWorker:1137/ThreadPoolExecutor$Worker.run:637}
08-12 13:56:19.591 16032 25901 D cgeo    : [OkHttp] 304 [Not Modified] (13 ms) GET https://www.opencaching.de/images/uploads/4D9C9129-2ABB-11EE-8B48-CDF35CEF9D44.jpg (http/1.1) {Network$LoggingInterceptor.intercept:412/RealInterceptorChain.proceed:109/Network$HeadersInterceptor.intercept:390/RealInterceptorChain.proceed:109/RealCall.getResponseWithInterceptorChain$okhttp:201/RealCall$AsyncCall.run:517/ThreadPoolExecutor.runWorker:1137/ThreadPoolExecutor$Worker.run:637}
08-12 13:56:19.591 16032 25898 D cgeo    : [refresh--1] Saving OC17B2E Auf Safari: Arbeitersiedlungen ([16, 66, 20]) to DB {DataStore.storeIntoDatabase:2220/DataStore.saveCaches:2182/DataStore.saveCache:2118/Geocache.storeCache:2120/Geocache.refreshSynchronous:2027/Geocache.refreshSynchronous:2021/Geocache.lambda$refresh$4:2017/Geocache.$r8$lambda$DZTiYZWuYZA0hxIqUWOy-0OdhP8:0}
08-12 13:56:19.591 16032 25898 D cgeo    : [refresh--1] _AL getAlcMode: 0 {Geocache.getAlcMode:1289/DataStore.storeIntoDatabase:2276/DataStore.saveCaches:2182/DataStore.saveCache:2118/Geocache.storeCache:2120/Geocache.refreshSynchronous:2027/Geocache.refreshSynchronous:2021/Geocache.lambda$refresh$4:2017}
08-12 13:56:19.601 16032 16032 D cgeo    : [main] [CtxLog]DataStore.loadCachesFromGeoCodes(#1):START {DataStore.loadCachesFromGeocodes:2810/DataStore.loadCaches:2756/DataStore.loadCache:2720/SearchResult.getFirstCacheFromResult:281/CacheDetailActivity.notifyDataSetChanged:893/CacheDetailActivity.notifyDataSetChanged:2894/CacheDetailActivity.access$5000:195/CacheDetailActivity$RefreshCacheHandler.handleRegularMessage:2874}
08-12 13:56:19.601 16032 16032 D cgeo    : [main] _AL setAlcMode: 0 {Geocache.setAlcMode:1284/DataStore.createCacheFromDatabaseContent:2971/DataStore.loadCachesFromGeocodes:2836/DataStore.loadCaches:2756/DataStore.loadCache:2720/SearchResult.getFirstCacheFromResult:281/CacheDetailActivity.notifyDataSetChanged:893/CacheDetailActivity.notifyDataSetChanged:2894}
08-12 13:56:19.602 16032 16032 D cgeo    : [main] [CtxLog]DataStore.loadCachesFromGeoCodes(#1):END (2ms)flags:[DB_MINIMAL, ATTRIBUTES, WAYPOINTS, SPOILERS, LOGS, INVENTORY, OFFLINE_LOG](1ms);gc#1[OC17B2E](1ms);RET:#1(2ms); {DataStore.loadCachesFromGeocodes:2893/DataStore.loadCaches:2756/DataStore.loadCache:2720/SearchResult.getFirstCacheFromResult:281/CacheDetailActivity.notifyDataSetChanged:893/CacheDetailActivity.notifyDataSetChanged:2894/CacheDetailActivity.access$5000:195/CacheDetailActivity$RefreshCacheHandler.handleRegularMessage:2874}
08-12 13:56:19.653 16032 16032 D cgeo    : [main] [CtxLog]forEachImageUrlInHtml:START {ImageUtils.forEachImageUrlInHtml:498/Geocache.getFilteredSpoilers:989/CacheDetailActivity$DescriptionViewCreator.setContent:1756/TabbedViewPagerFragment.notifyDataSetChanged:63/TabbedViewPagerActivity.setOrderedPages:119/CacheDetailActivity.notifyDataSetChanged:910/CacheDetailActivity.notifyDataSetChanged:2894/CacheDetailActivity.access$5000:195}
08-12 13:56:19.654 16032 16032 D cgeo    : [main] [CtxLog]forEachImageUrlInHtml:END (1ms)size:2669(1ms);#found:1(1ms); {ImageUtils.forEachImageUrlInHtml:525/Geocache.getFilteredSpoilers:989/CacheDetailActivity$DescriptionViewCreator.setContent:1756/TabbedViewPagerFragment.notifyDataSetChanged:63/TabbedViewPagerActivity.setOrderedPages:119/CacheDetailActivity.notifyDataSetChanged:910/CacheDetailActivity.notifyDataSetChanged:2894/CacheDetailActivity.access$5000:195}
08-12 13:56:19.657 16032 16032 D cgeo    : [main] [CtxLog]forEachImageUrlInHtml:START {ImageUtils.forEachImageUrlInHtml:498/ImageUtils.addImagesFromHtml:481/Geocache.getImages:2204/Geocache.getNonStaticImages:2215/CacheDetailActivity$ImageGalleryCreator.setContent:2413/TabbedViewPagerFragment.notifyDataSetChanged:63/TabbedViewPagerActivity.setOrderedPages:119/CacheDetailActivity.notifyDataSetChanged:910}
08-12 13:56:19.657 16032 16032 D cgeo    : [main] [CtxLog]forEachImageUrlInHtml:END (0ms)size:2669(0ms);#found:1(0ms); {ImageUtils.forEachImageUrlInHtml:525/ImageUtils.addImagesFromHtml:481/Geocache.getImages:2204/Geocache.getNonStaticImages:2215/CacheDetailActivity$ImageGalleryCreator.setContent:2413/TabbedViewPagerFragment.notifyDataSetChanged:63/TabbedViewPagerActivity.setOrderedPages:119/CacheDetailActivity.notifyDataSetChanged:910}
08-12 13:56:19.714 16032 16032 D cgeo    : [main] [CtxLog]forEachImageUrlInHtml:START {ImageUtils.forEachImageUrlInHtml:498/Geocache.getFilteredSpoilers:989/CacheDetailActivity$DescriptionViewCreator.setContent:1756/TabbedViewPagerFragment.notifyDataSetChanged:63/TabbedViewPagerActivity.reinitializeViewPager:235/CacheDetailActivity.notifyDataSetChanged:911/CacheDetailActivity.notifyDataSetChanged:2894/CacheDetailActivity.access$5000:195}
08-12 13:56:19.714 16032 16032 D cgeo    : [main] [CtxLog]forEachImageUrlInHtml:END (0ms)size:2669(0ms);#found:1(0ms); {ImageUtils.forEachImageUrlInHtml:525/Geocache.getFilteredSpoilers:989/CacheDetailActivity$DescriptionViewCreator.setContent:1756/TabbedViewPagerFragment.notifyDataSetChanged:63/TabbedViewPagerActivity.reinitializeViewPager:235/CacheDetailActivity.notifyDataSetChanged:911/CacheDetailActivity.notifyDataSetChanged:2894/CacheDetailActivity.access$5000:195}
08-12 13:56:19.758 16032 16032 I MSHandlerLifeCycle: removeMultiSplitHandler: no exist. decor=DecorView@879ef8c[Aktualisieren]
08-12 13:56:19.759 16032 16091 D OpenGLRenderer: setSurface called with nullptr
08-12 13:56:19.759 16032 16091 D OpenGLRenderer: setSurface() destroyed EGLSurface
08-12 13:56:19.759 16032 16091 D OpenGLRenderer: destroyEglSurface
08-12 13:56:19.760 16032 16032 I ViewRootImpl@ed1de90[Aktualisieren]: dispatchDetachedFromWindow
08-12 13:56:19.768 16032 16032 D InputTransport: Input channel destroyed: 'f50765c', fd=136
08-12 13:56:19.817 16032 16032 D cgeo    : [main] LooperLogger: total:#4000/7960ms/avg 1ms 
```

**Comment by eddiemuc** [comment]
Should be fixed with PR #14503
